gbpservice/neutron/nfp/configurator/api/v1/controllers/__init__.py

```diff
@@ -0,0 +1,42 @@
+from wsmeext import pecan as wsme_pecan
+from wsme import types as wtypes
+
+import controller
+
+"""this class call methods of Controller class
+according to HTTP request"""
+
+
+class ControllerResolver(object):
+
+    create_network_function_device_config = controller.Controller(
```

Comment:
Variable inside init() function is treated as instance variable and outside init() is treated as class variable. For invoking controller according to path pecan needs this variable as class variable.